client/app/components/ui/help-text.hbs

```diff
@@ -0,0 +1,3 @@
+<p class="text-small tiny-margin-bottom" ...attributes>
```

Comment:
I wonder if help-text can just be a class

Reply:
`.help-text` already is a class defined in our style guide. But (per the customer's request) we are not following the [style guide's recommendation](https://nyc-planning-style-guide.netlify.app/forms/#help-text) (in which help text is below the input). We'd need to come up with a more semantic class name for this custom thing we're doing — like `.inline-guidance` or `.sub-label`… or maybe modify `.help-text` with a second class like `.help-text.above`.

Reply:
Oh, I see. Though, I guess guides are just guides in the end and we have a customer request :p so maybe using it above an input is okay? I like the `.help-text.above` idea!